newIDE/app/src/MainFrame/EditorContainers/HomePage/GetStartedSection/index.js

```diff
@@ -628,16 +628,10 @@ const GetStartedSection = ({
     return (
       <>
         <SectionContainer
-          title={
-            profile && profile.username ? (
-              <Trans>Hello {profile.username}!</Trans>
```

Comment:
I guess your thinking is that the username can be unexpected + already displayed in the profile button?

Reply:
Yes, basically when I launch GDevelop for the first time, the screen is saying me "Hello" and not telling what it's going to help me to do much. If I click on the button to create an account and do it, I'm not back at this screen and it says "Hello + my username". Great but I know who I am, so better make this first screen very clear: "You're going to start making games, the title say so and you have big buttons to have you follow a tutorial" "When you want to explore something else, the only alternative is to click the bottom buttons, where you'll find the build page which is the second one, so the 'obvious' next step.".